package-lock.json

```diff
@@ -0,0 +1,5510 @@
+{
+  "name": "react-image-crop",
+  "version": "2.0.8",
```

Comment:
I noticed the lock file doesn't have 2.1.0, not too familiar with lockfiles but I'm guessing it should?